docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-microsoft-365-exchange-dkim-signing-configuration-disabled.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[prebuilt-rule-0-16-2-microsoft-365-exchange-dkim-signing-configuration-disabled]]
+=== Microsoft 365 Exchange DKIM Signing Configuration Disabled
+
+Identifies when a DomainKeys Identified Mail (DKIM) signing configuration is disabled in Microsoft 365. With DKIM in Microsoft 365, messages that are sent from Exchange Online will be cryptographically signed. This will allow the receiving email system to validate that the messages were generated by a server that the organization authorized and were not spoofed.
```

Comment:
```suggestion
Identifies when the DomainKeys Identified Mail (DKIM) signing configuration is disabled in Microsoft 365. With DKIM in Microsoft 365, messages that are sent from Exchange Online mail will be cryptographically signed. This will allow the receiving email system to validate that the messages were generated by a server that the organization authorized and were not spoofed.
```